ramls/circulation.raml

```diff
@@ -278,8 +278,12 @@ resourceTypes:
             description: "Patron type id"
             pattern: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
             required: true
-          shelving_location_id:
-            description: "Shelving location id"
+          location_id:
+            description: "Location id"
+            pattern: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
+            required: true
+          institution_id:
+            description: "Institution id"
```

Comment:
The location_id parameter rename is ok.
Why do you also require the client to also pass in the institution_id to the API? CirculationPolicyRepository.java uses the locationsStorageClient to fetch the Location.java record that includes the institution_id.
There are two possibilities:
A) Require the institution_id in the API, then we should remove fetching the Loction.java record.
B) Remove the institution_id from the API and use the institution_id from the fetched Location.java record.

Reply:
@julianladisch So, the only thing what I need to do(regarding current version of code in this PR) - is to remove "institution_id" from 'circulation.raml'.
Am I right?

Reply:
@julianladisch I want to add, that if "institution_id" won`t be passed as a parameter in the request at the 'CirculationRulesClient.java' - then there will be no opportunity to fetch this "institution_id"  via "location_id" at the 'Drools.java' (where these ids received from MultiMap). Please, correct me if I`m wrong.